cmd/stellar-rpc/internal/integrationtest/infrastructure/test.go

```diff
@@ -136,6 +146,11 @@ func NewTest(t *testing.T, cfg *TestConfig) *Test {
 			shouldWaitForRPC = !cfg.OnlyRPC.DontWait
 		}
 		parallel = !cfg.NoParallel
+		i.enableCoreHTTPQueryServer = cfg.EnableCoreHTTPQueryServer
+	}
+
+	if i.enableCoreHTTPQueryServer && GetCoreMaxSupportedProtocol() < 22 {
```

Comment:
We should change this to 23 once there's a testing build of 23

Reply:
Yep!